npm.js

```diff
@@ -62,12 +62,16 @@ exports.translate = function(load){
 				packages[pkg.name+"@"+pkg.version] = true;
 			}
 		});
-		var configDependencies = ['@loader','npm-extension'].concat(configDeps.call(loader, pkg));
+		var configDependencies = [
+			'@loader',
+			'npm-extension',
+			'process'
```

Comment:
What I mean is that process is an npm dependency, right?  If so it requires the npm extension to be loaded to work, and the `configDependencies` you are adding process to here are run before the npm extension. So process will not load correctly.  It's running in your tests because you are setting a path for it.

> It's not obvious to me why we'd opt-out of being compatible when we already know that some npm packages use Node's global process object. Why wouldn't we support those packages if a comprehensive list of requirements is available and a proven implementation is accessible?

Because it's almost 150 LOC that would be included always (including production builds) even in projects that don't need it.